### PR TITLE
console -> console-socket

### PR DESCRIPTION
**Pull request by ehazlett**
Update to use runc `--console-socket`.

Signed-off-by: Evan Hazlett <ejhazlett@gmail.com>

### CODE REVIEW

File: runc.go [hunk 1]
```diff
@@ -108,7 +108,7 @@ func (o *CreateOpts) args() (out []string) {
 		out = append(out, "--pid-file", o.PidFile)
 	}
 	if o.Console != "" {
-		out = append(out, "--console", o.Console)
+		out = append(out, "--console-socket", o.Console)
 	}
```
Comment (AkihiroSuda) [inline runc.go]: Probably o.Console should be renamed to o.ConsoleSocket?

**Comment by ehazlett** [comment]
Good point. I'll update. Thx

**Comment by AkihiroSuda** [comment]
Also, it would be great if the supported version of runc is clarified in the document

**Comment by AkihiroSuda** [comment]
Any update?

**Comment by ehazlett** [comment]
Updated references to `Console` -> `ConsoleSocket`.  Added minimum version of runc in the readme.  thx!

**Pull request merged**